### PR TITLE
added list crud function

### PR DESCRIPTION
**Pull request by pegahnatanzi**
Kelly and I completed the list customers story.

### CODE REVIEW

File: customers.code-workspace [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (kellyayuen) [inline customers.code-workspace:1]: @pegahnatanzi Did we want to push this for everyone else?

Comment (pegahnatanzi) [reply customers.code-workspace:1]: @kellyayuen I think it could be helpful! so doesn't hurt. what do you think?

Comment (kellyayuen) [reply customers.code-workspace:1]: Ok sounds good! I was just checking :)

**Comment by kellyayuen** [comment]
@pegahnatanzi The code looks good! Merging this pull request

**Pull request merged**